apps/api/.env.example

```diff
@@ -4,6 +4,9 @@ SERVER_PORT=
 # Node env
 NODE_ENV= # Use "development" for graphql playground to work
 
+# Upload folder location
+Upload_Path=./src/uploads/
```

Comment:
This is incorrect. We need to provide full path here. Not relative path.
Also that variable is inconsistent with others

Reply:
Updated